core/rpc/Cargo.toml

```diff
@@ -8,7 +8,8 @@ edition = "2018"
 api = { package = "substrate-rpc-api", path = "./api" }
 client = { package = "substrate-client", path = "../client" }
 codec = { package = "parity-scale-codec", version = "1.0.0" }
-futures03 = { package = "futures-preview", version = "0.3.0-alpha.19", features = ["compat"] }
+futures = { version = "0.3.1", features = ["compat"] }
+futures01 = { package = "futures", version = "0.1.29" }
```

Comment:
That was a `dev-dependency` before, is there a reason for moving it to an actual dependency?

Reply:
Nope, fixed!